locales-pending/premium.ftl

```diff
@@ -170,7 +170,7 @@ modal-exposure-status-title = About exposure statuses
 modal-exposure-status-description = We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info.
   Your exposures will have one of the following statuses:
 modal-exposure-status-action-needed = <b>Action needed</b> means it is currently active and you need to take steps to fix it.
-modal-exposure-status-in-progress = <b>In progress</b> means we are actively working on fixing the exposure for you. This is a { -brand-premium } feature.
+modal-exposure-status-in-progress = <b>In progress</b> means we are actively working on fixing the exposure for you. This is a { -brand-monitor-plus } feature.
 modal-exposure-status-fixed = <b>Fixed</b> means the exposure has been resolved and theres no action for you to take.
```

Comment:
Unrelated, but now my eyes are confused... Should this be "there’s" vs "theres"?
Or just expand to "there is" and avoid my apostrophe question altogether.

Looks like perhaps the only instance:
```sh
git grep -in "ThErEs" -- "*.ftl" | cat

locales-pending/premium.ftl:174:modal-exposure-status-fixed = <b>Fixed</b> means the exposure has been resolved and theres no action for you to take.
```